### PR TITLE
Add Dockerfile for arm64.

### PR DESCRIPTION
**Pull request by TrevorTaoARM**
Add Dockerfile of projectcalico/go-build to support arm64 platform

### CODE REVIEW

**Comment by TrevorTaoARM** [comment]
This Dockerfile for arm64 has been verified with projectcalico/cni-plugin

**Comment by caseydavenport** [comment]
@fasaxc would you be a good reviewer for this as well?

**Comment by djlwilder** [comment]
Should the Makefile be update to add a make target calico/go-build-arm64 ?

**Comment by TrevorTaoARM** [comment]
@djlwilder Yes, I would update the Makefile to add a make target  calico/go-build-arm64.
Thanks.

**Comment by TrevorTaoARM** [comment]
@djlwilder Makefile updated

**Comment by TrevorTaoARM** [comment]
@fasaxc I would answer you soon

**Comment by TrevorTaoARM** [comment]
@fasaxc Yes, it'ok, I would submit a new patch with my email included.

File: Dockerfile.arm64 [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  && gometalinter --install
+
+# Install license checking tool
+RUN go get github.com/pmezard/licenses
+
+# Install tool to merge coverage reports
+RUN go get github.com/wadey/gocovmerge
+
+# Install patched version of goveralls (upstream is bugged if not used from Travis)
+RUN wget https://github.com/fasaxc/goveralls/releases/download/v0.0.1-smc/goveralls \
```
Comment (fasaxc) [inline Dockerfile.arm64]: This binary is the wrong arch for ARM64.  Take a look at David Wilder's recent patch to build the patched version.

**Comment by TrevorTaoARM** [comment]
@fasaxc Thank you, I add the 'goveralls' for arm64 from github src,

**Comment by TrevorTaoARM** [comment]
I submitted a new PR #20
https://github.com/projectcalico/go-build/pull/20
It has resolved the conflicts, so this PR would be closed

**Pull request closed without merging**